.ci/docker/ubuntu-cuda/Dockerfile

```diff
@@ -134,6 +134,8 @@ COPY ./common/install_cudnn.sh install_cudnn.sh
 RUN if [ "${CUDNN_VERSION}" -eq 8 ]; then bash install_cudnn.sh; fi
 RUN rm install_cudnn.sh
 
+# Install torch inductor benchmark deps
```

Comment:
You would need to copy all the commit pins above into the Docker images for the script to work.  I have an example PRs for triton here https://github.com/pytorch/pytorch/pull/95233 and https://github.com/pytorch/pytorch/pull/95754